packages/slate/test/interfaces/Editor/positions/range/inline-reverse.tsx

```diff
@@ -0,0 +1,44 @@
+/** @jsx jsx */
```

Comment:
I felt this test was missing (in keeping with the other existing tests). Including this helped correct a bad assumption on my part too, while trying to understand the issue, so I felt it might be good to include it here as well.